airflow/www/views.py

```diff
@@ -4589,7 +4591,10 @@ def varimport(self, session):
                     skipped.add(k)
                     continue
                 try:
-                    models.Variable.set(k, v, serialize_json=not isinstance(v, str))
+                    if isinstance(v,dict) and "description" in v and "val" in v:
+                        models.Variable.set(k, v["val"], description=v["description"], serialize_json=not isinstance(v["val"], str))
+                    else:
+                        models.Variable.set(k, v, serialize_json=not isinstance(v, str))
```

Comment:
Instead of needing this if-else, it’s probably better to always make `v` a dict instead.

Reply:
this is only to support backward compatibility for import. export always uses value as dict